x-pack/plugins/spaces/index.ts

```diff
@@ -145,8 +145,17 @@ export const spaces = (kibana: any) =>
       registerUserProfileCapabilityFactory(async request => {
         const spacesClient = server.plugins.spaces.spacesClient.getScopedClient(request);
 
+        let manageSecurity = false;
+
+        if (server.plugins.security) {
+          const { allowRbac = false } =
+            xpackMainPlugin.info.feature('security').getLicenseCheckResults() || {};
+          manageSecurity = allowRbac;
```

Comment:
Can we switch this to use `showLinks` from the security license check results, as this is what is driving whether the security role/user links are shown on the management screens, which seems to be the same thing that we're checking here, just from a different place.